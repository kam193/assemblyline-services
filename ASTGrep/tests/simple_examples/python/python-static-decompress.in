import zlib
from zlib import decompress

zlib.decompress(b'x\x9cKLL\x04\x00\x02I\x01$')
decompress(b"x\x9cKLL\x04\x00\x02I\x01$")

import lzma
lzma.decompress(b'\xfd7zXZ\x00\x00\x04\xe6\xd6\xb4F\x02\x00!\x01\x16\x00\x00\x00t/\xe5\xa3\x01\x00\x02aaa\x00\x00k\xc5\x16W\xab\xb5g%\x00\x01\x1b\x03\x0b/\xb9\x10\x1f\xb6\xf3}\x01\x00\x00\x00\x00\x04YZ')

import bz2
bz2.decompress(b'BZh91AY&SY=L3K\x00\x00\x00\x01\x00<\x00 \x00!\x9ah3M\x13<]\xc9\x14\xe1B@\xf50\xcd,')
