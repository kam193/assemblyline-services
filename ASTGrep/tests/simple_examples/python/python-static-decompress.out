import zlib
from zlib import decompress

b'aaa'
b'aaa'

import lzma
b'aaa'

import bz2
b'abcd'
